package/libs/libnetfilter-queue/Makefile

```diff
@@ -9,13 +9,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libnetfilter_queue
 PKG_VERSION:=1.0.2
```

Comment:
if you point it at  arev newer than 1.0.2 then the version string needs to change